src/Network/Mattermost/WebSocket.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
Also, bear in mind that we tend not to use this extension much unless the benefit is significant. (Here it is minimal.)